src/applications/disability-benefits/all-claims/tests/schema/transformedData.js

```diff
@@ -256,6 +255,172 @@ export const transformedMaximalData = {
   },
 };
 
+export const transformedNewSecondaryData = {
```

Comment:
So...this is fine for now, but heads up: I did change the way this works in https://github.com/department-of-veterans-affairs/vets-website/pull/9550. Basically, each of these transformed data sets will get its own JSON file in a separate directory.